.github/workflows/pages.yml

```diff
@@ -0,0 +1,82 @@
+# Workflow for building and deploying a the /docs folder as static Jekyll site to GitHub Pages
+name: Deploy Jekyll site to Pages
+
+on:
+  push:
+    branches:
+      - "main"
+      - "markdown-documentation"
+    paths:
```

Comment:
Add the following paths to this list:

- This workflow: `.github/workflows/pages.yml`
- Container image descriptor directory: `docker/document-site-builder/**`
- Build script: `tests/build-documentation-site.sh`
- Common script: `scripts/common.sh`

If you don't want to be that granular, you can remove this list altogether.

Reply:
Done